doc/interfaces/data_fetch.rst

```diff
@@ -0,0 +1,23 @@
+-------------------------
+Downloading DIPY datasets
+-------------------------
+In this tutorial, you will learn how to download the DIPY datasets using the terminal.
+
+First, let's create a folder::
+
+    mkdir data_folder
+
+You can list all available datasets in DIPY using the command::
+
+    dipy_fetch list
+
+In order to fetch a specific dataset, you will specify its name with the ``dipy_fetch``
+command, and an optional destination directory using the ``--out_dir`` argument::
```

Comment:
Sorry, now reading this again I realize that my wording was not too appropriate. Sorry. I think this is better: 

In order to fetch a specific dataset, you will need to specify its name to the {double_backticks}dipy_fetch{double_backticks}
command, and an optional destination directory using the {double_backticks}--out_dir{double_backticks} argument::

Reply:
Reworded in 8899fa6